docs/getting-started/devenv/using-visual-studio-code.md

```diff
@@ -1,16 +1,378 @@
+Core Solution using [Visual Studio Code](http://codde.visualstudio.com).
+You'll learn how to create projects, create unit tests, use the debugging
+tools, and incorproate third-party libraries via [NuGet](http://nuget.org).
+
+This article uses Visual Studio Code on Mac OS. Where there are differences,
+it points out the differences for the Windows platform.
+
+Prerequisites
+-------------
+
+Before starting, you'll need to install the [.NET Core SDK](https://www.microsoft.com/net/download),
+currently in a preview relese. The .NET Core SDK includes the latest release
+of the .NET Core framework and runtime.
+
+You'll also need to install [Visual Studio Code](http://code.visualstudio.com).
+During the course of this article, you'll also install extensions
+that will improve the .NET Core development experience.
+
+You can find the links to all of these at the [.NET home page](http://dot.net).
+
+Getting Started
+---------------
+
+The source for this tutorial is available on
+[GitHub](https://github.com/dotnet/core-docs/tree/master/samples/getting-started/golden).
+
+Start Visual Studio Code. Press Ctrl + '`' (the back-quote character) to open
+an embedded terminal in VS Code. (Alternatively, you can use a separate
+terminal window, if you prefer).
+
+By the time we're done, you'll create three projects: a library project,
+tests for that library project, and a console application that makes
+use of the library. You'll follow a standard folder structure for
+the three projects. Following this standard folder structure
+maeans that the .NET Core SDK tools understand the relationship between
+your production code projects and your test code projects. That makes
+your development exerience more productive.
+
+Let's start by creating those folders. In the terminal, create a 'golden'
+directory. Under that directory create `src` and `test`
+directories. Under `src` create `app` and `library` directories. In `test`
+create a `test-library` directory. You can do this either using the terminal
+in VS code, or by clicking on the parent folder in VS Code and selecting the
+"New Folder" icon.
+
+In VS Code, open the 'golden' directory. This directory is the root of your solution.
+
+Next, create a `global.json` file in the root directory for your solution.
+The contents of `global.json` are:
+
+```js
+{
+    projects: [
+        "src",
+        "test"
+    ]
+}
+```
+
+At this point, your directory tree should look like this:
+
+
+```
+/golden
+|__global.json
+|__/src
+   |__/app
+   |__/library
+/test
+   |__/test-library
+```
+
+### Writing the library
+
+Your next task is to create the library. In the terminal window
+(either the embedded terminal in VS code, or another terminal),
+cd to `golden/src/library` and type the command `dotnet new`.
+
+The `dotnet new` command creates a new application project. Because
+you want a library project, you're going to make a few changes to
+the generated `project.json` file.
```

Comment:
Since we are nearing the RTM and this is not merged, maybe we fix it now to use `dotnet new -t lib`?